.gitignore

```diff
@@ -1,4 +1,5 @@
-bin
+host-api
```

Comment:
Ultimately, shoiuld pull this from an environment variable, like python-agent. I'd like to keep it like this for now and put in a separate issue for that.